### ISSUE TITLE
Email verification for UB students

### ISSUE THREAD

**Issue by edentron** [cse442-fall-2019-offering/442projects-loopholes#6]
As a UB student, I want to verify that I attend UB with my email so that I can access the page's locked features (templates and upload).

**Comment by supratikn** [comment]
Yeah, just need their emails for posting. We can set one vote per ip address.
